website/docs/d/organization_custom_role.html.markdown

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
Can you please update the `github.erb` file with links to these awesome new docs?

Reply:
@kfcampbell yep added new commit. thanks